plugins/tactics/src/Ide/Plugin/Tactic.hs

```diff
@@ -192,19 +193,19 @@ provide tc name _ plId uri range _ = do
 -- predicate holds for the goal.
 requireExtension :: Extension -> TacticProvider -> TacticProvider
 requireExtension ext tp dflags plId uri range jdg =
-  case xopt ext dflags of
-    True  -> tp dflags plId uri range jdg
-    False -> pure []
+  if xopt ext dflags
+    then tp dflags plId uri range jdg
+    else pure []
```

Comment:
The use of "case ... True ... False" is consistent and obviously deliberate in the tactics plugin. Should we just ignore the use case suggestion for this module and leave the code alone?

(I don't care either way, I basically made an arbitrary decision just to get the PR started.)